feature-libs/cart/base/core/facade/multi-cart.service.spec.ts

```diff
@@ -324,24 +330,131 @@ describe('MultiCartService', () => {
   });
 
   describe('mergeToCurrentCart', () => {
-    it('should merge cart', () => {
-      spyOn(service as any, 'generateTempCartId').and.returnValue('temp-uuid');
-
-      service.mergeToCurrentCart({
-        userId: 'userId',
-        cartId: 'cartId',
-        extraData: {},
+    describe('feature flag incrementProcessesCountForMergeCart is enabled', () => {
+      it('should merge cart', () => {
+        spyOn(service as any, 'generateTempCartId').and.returnValue(
+          'temp-uuid'
+        );
+        const isEnabledSpy = featureConfigService.isEnabled as jasmine.Spy;
+
+        // incrementProcessesCountForMergeCart is enabled
+        isEnabledSpy.and.callFake((value: string) => {
+          if (value === 'incrementProcessesCountForMergeCart') {
+            return true;
+          }
+          return false;
+        });
+        service.mergeToCurrentCart({
+          userId: 'userId',
+          cartId: 'cartId',
+          extraData: {},
+        });
+        expect(store.dispatch).toHaveBeenCalledWith(
+          new CartActions.MergeCartAndIncrementProcessesCount({
+            userId: 'userId',
+            extraData: {},
+            cartId: 'cartId',
+            tempCartId: 'temp-uuid',
+          })
+        );
       });
-
-      expect(store.dispatch).toHaveBeenCalledWith(
-        new CartActions.MergeCart({
+    });
+    describe('feature flag authorizationCodeFlowByDefault is enabled', () => {
+      it('should merge cart', () => {
+        spyOn(service as any, 'generateTempCartId').and.returnValue(
+          'temp-uuid'
+        );
+        const isEnabledSpy = featureConfigService.isEnabled as jasmine.Spy;
+
+        // authorizationCodeFlowByDefault is enabled
```

Comment:
we have that information in describe block

what are purposes of remaining commented code below?

Reply:
Good point! These are leftovers used before I decidec to move tests to separate blocks.